packages/core/src/index.js

```diff
@@ -2,7 +2,16 @@
 export { default as A } from "./A";
 export { default as DocumentsAndDatasets } from "./DocumentsAndDatasets";
 export { default as Filter } from "./Filter";
-export { default as Footer } from "./Footer";
+export {
+  default as Footer,
+  AboutOrganization,
+  AboutInitiative,
+  Initiative,
+  OwnerLogo,
+  QuickLinks,
+  LegalLinks,
+  StayInTouch,
+} from "./Footer";
```

Comment:
I think all of these should be exported with `Footer` suffix e.g. `FooterInitiative`, `FooterStayInTouch`, etc., just like when we import `Card`, `CardActionArea`, etc., from `@material-ui/core`... I believe it should still be possible to import `@commons-ui/core/Footer/StayInTouch` directly if somebody wished to name it/import it differently.

May be https://github.com/CodeForAfrica/COMMONS-UI/pull/37/files#diff-a7da307727085aae48f690b293c9c1e9R35-R39 is where we should rename them.